docs/main_text.md

```diff
@@ -60,31 +60,148 @@ If you would prefer to report issues via e-mail, you can also email [ccdl@alexsl
 
 ## refine.bio processed
 
-### RNA-seq pipelines
+Because refine.bio is designed to be consistently updated, we use processing and normalization methods that operate on a single sample wherever possible.
+Processing and normalization methods that require multiple samples generally have outputs that are influenced by whatever samples are included and rerunning these methods whenever a new sample is added to the system would be impractical.
```

Comment:
Most people may not know the difference between single sample and multiple-samples. Do you want to give an example (RMA)?